src/qtt/utilities/tools.py

```diff
@@ -1142,7 +1148,6 @@ def addPPTslide(title=None, fig=None, txt=None, notes=None, figsize=None,
         warnings.warn(
             'addPPTslide is not available on your system. Install win32com from https://pypi.org/project/pypiwin32/.')
```

Comment:
Python (mypy) is confused about which addPPTslide function to use.

Reply:
@bogdanvconstantin The problem here is that it is a conditional function (it is in a try except ImportError construction) depending on if you are running windows, which you aren't running. The parameters of both functions should at least be the same, which they aren't currently. mypy is complaining about that.

@peendebak you should have noticed the build failed (and fix of course) before asking for a review.

Reply:
Point taken